libvast/builtins/endpoints/query.cpp

```diff
@@ -720,10 +723,13 @@ class plugin final : public virtual rest_endpoint_plugin {
     return endpoints;
   }
 
-  system::rest_handler_actor
-  handler(caf::actor_system& system, system::node_actor node) const override {
-    return system.spawn(request_multiplexer, node);
+  auto handler(caf::actor_system& system, system::node_actor node) const
+    -> system::rest_handler_actor override {
+    return system.spawn(request_multiplexer, node, &config_);
```

Comment:
While technically safe to do as the plugin is a singleton, can you change the config to be passed by value and kept in the state of the request multiplexer as a hard copy? Otherwise, if we ever want to support loading/unloading plugins at runtime this will become a problem.